src/js/views/channel_slider_view.js

```diff
@@ -104,11 +104,12 @@ var ChannelSliderView = Backbone.View.extend({
                 var endAvg = parseInt(ends.reduce(addFn, 0) / ends.length, 10);
                 var startsNotEqual = starts.reduce(allEqualFn, starts[0]) === false;
                 var endsNotEqual = ends.reduce(allEqualFn, ends[0]) === false;
-                var min = mins.reduce(reduceFn(Math.min), 9999);
-                var max = maxs.reduce(reduceFn(Math.max), -9999);
+                var min = mins.reduce(reduceFn(Math.min));
+                var max = maxs.reduce(reduceFn(Math.max));
+                console.log(min, max);
```

Comment:
`console.log` tend to be removed, right? Have you looked into a longer-term debugging strategy? Likely depends on the framework of course but https://stackoverflow.com/questions/94934/what-debug-logging-tools-are-available-from-javascript